### DISCUSSION TITLE
telnet: bad address 'tcp-echo-cloud-svc.cloudzone'

### DISCUSSION THREAD

**Discussion by Aix6** [kubeedge/kubeedge#3847] [Q&A]
Using the following test case:
https://edgemesh.netlify.app/guide/test-case.html#cross-edge-cloud

When I deploy the "Cloud access edge " test case, they are scheduled correctly. But when I try using the "Edge access cloud" test case, I get the error "telnet: bad address 'tcp-echo-cloud-svc.cloudzone'".

the log from the edge agent:

[INFO] 172.17.0.2:59781 - 2 "AAAA IN tcp-echo-cloud-svc.cloudzone. udp 46 false 512" NXDOMAIN qr,rd,ra 46 0.011376754s
[INFO] 172.17.0.2:45856 - 3 "AAAA IN tcp-echo-cloud-svc.cloudzone.edgezone.svc.cluster.local. udp 73 false 512" SERVFAIL qr,aa,rd 166 0.00017384s
[INFO] 172.17.0.2:55856 - 4 "AAAA IN tcp-echo-cloud-svc.cloudzone.svc.cluster.local. udp 64 false 512" SERVFAIL qr,aa,rd 157 0.000176988s
[INFO] 172.17.0.2:48115 - 5 "AAAA IN tcp-echo-cloud-svc.cloudzone.cluster.local. udp 60 false 512" SERVFAIL qr,aa,rd 153 0.000101509s
[INFO] 172.17.0.2:59521 - 6 "AAAA IN tcp-echo-cloud-svc.cloudzone. udp 46 false 512" NXDOMAIN qr,rd,ra 46 0.012526947s
[INFO] 172.17.0.2:43397 - 7 "AAAA IN tcp-echo-cloud-svc.cloudzone.edgezone.svc.cluster.local. udp 73 false 512" SERVFAIL qr,aa,rd 166 0.000090932s
[INFO] 172.17.0.2:42690 - 8 "AAAA IN tcp-echo-cloud-svc.cloudzone.svc.cluster.local. udp 64 false 512" SERVFAIL qr,aa,rd 157 0.00005627s
[INFO] 172.17.0.2:52825 - 9 "AAAA IN tcp-echo-cloud-svc.cloudzone.cluster.local. udp 60 false 512" SERVFAIL qr,aa,rd 153 0.000055028s
[INFO] 172.17.0.2:58870 - 10 "AAAA IN tcp-echo-cloud-svc.cloudzone. udp 46 false 512" NXDOMAIN qr,rd,ra 46 0.010677519s
[INFO] 172.17.0.2:49946 - 11 "AAAA IN tcp-echo-cloud-svc.cloudzone.edgezone.svc.cluster.local. udp 73 false 512" SERVFAIL qr,aa,rd 166 0.000089105s
[INFO] 172.17.0.2:48067 - 12 "AAAA IN tcp-echo-cloud-svc.cloudzone.svc.cluster.local. udp 64 false 512" SERVFAIL qr,aa,rd 157 0.000129666s
[INFO] 172.17.0.2:44724 - 13 "AAAA IN tcp-echo-cloud-svc.cloudzone.cluster.local. udp 60 false 512" SERVFAIL qr,aa,rd 153 0.00009314s
[INFO] 172.17.0.2:55959 - 14 "A IN tcp-echo-cloud-svc.cloudzone. udp 46 false 512" NXDOMAIN qr,rd,ra 46 0.013949858s
[INFO] 172.17.0.2:38798 - 15 "A IN tcp-echo-cloud-svc.cloudzone.edgezone.svc.cluster.local. udp 73 false 512" SERVFAIL qr,aa,rd 166 0.000161544s
[INFO] 172.17.0.2:59545 - 16 "A IN tcp-echo-cloud-svc.cloudzone.svc.cluster.local. udp 64 false 512" SERVFAIL qr,aa,rd 157 0.000137076s
[INFO] 172.17.0.2:41051 - 17 "A IN tcp-echo-cloud-svc.cloudzone.cluster.local. udp 60 false 512" SERVFAIL qr,aa,rd 153 0.000135414s
[INFO] 172.17.0.2:44841 - 18 "A IN tcp-echo-cloud-svc.cloudzone. udp 46 false 512" NXDOMAIN qr,rd,ra 46 0.010543393s
[INFO] 172.17.0.2:57659 - 19 "A IN tcp-echo-cloud-svc.cloudzone.edgezone.svc.cluster.local. udp 73 false 512" SERVFAIL qr,aa,rd 166 0.000086273s
[INFO] 172.17.0.2:56533 - 20 "A IN tcp-echo-cloud-svc.cloudzone.svc.cluster.local. udp 64 false 512" SERVFAIL qr,aa,rd 157 0.000078826s
[INFO] 172.17.0.2:48386 - 21 "A IN tcp-echo-cloud-svc.cloudzone.cluster.local. udp 60 false 512" SERVFAIL qr,aa,rd 153 0.00006085s
[INFO] 172.17.0.2:42190 - 22 "A IN tcp-echo-cloud-svc.cloudzone. udp 46 false 512" NXDOMAIN qr,rd,ra 46 0.015018106s
[INFO] 172.17.0.2:36478 - 23 "A IN tcp-echo-cloud-svc.cloudzone.edgezone.svc.cluster.local. udp 73 false 512" SERVFAIL qr,aa,rd 166 0.00006208s
[INFO] 172.17.0.2:39979 - 24 "A IN tcp-echo-cloud-svc.cloudzone.svc.cluster.local. udp 64 false 512" SERVFAIL qr,aa,rd 157 0.000035293s
[INFO] 172.17.0.2:33445 - 25 "A IN tcp-echo-cloud-svc.cloudzone.cluster.local. udp 60 false 512" SERVFAIL qr,aa,rd 153 0.000033415s

**Comment by abhinavjha126** [comment]
i am facing same issue...please let me know

**Comment by chenqiangzhishen** [comment]
the same issue
